src/Resources/ResourceFactory.php

```diff
@@ -195,6 +196,54 @@ protected function serviceFactory(array &$properties)
             unset($properties['attributes']['transit_time']['max']);
         }
 
+        if (isset($properties['attributes']['regions_from'][0])) {
+            $regionFromAttribute = $properties['attributes']['regions_from'][0];
+
+            $countryCode = $regionFromAttribute['country_code'];
+            $regionCode = $regionFromAttribute['region_code'];
+
+            /** @var RegionInterface $regionFrom */
+            $regionFrom = $this->api->getRegions([
+                    'country_code' => $countryCode,
+                    'region_code'  => $regionCode,
+                ]
+            )->get()[0];
+
+            if(!$regionFrom){
+                throw new ResourceNotFoundException(
+                    'No region found for country code '.$countryCode.' and region code '. $regionCode
+                );
+            }
+
+            $service->setRegionFrom($regionFrom);
+
+            unset($properties['attributes']['regions_from']);
+        }
+
+        if (isset($properties['attributes']['regions_to'][0])) {
+            $regionToAttribute = $properties['attributes']['regions_to'][0];
+
+            $countryCode = $regionToAttribute['country_code'];
+            $regionCode = $regionToAttribute['region_code'];
+
+            /** @var RegionInterface $regionTo */
+            $regionTo = $this->api->getRegions([
+                    'country_code' => $countryCode,
+                    'region_code'  => $regionCode,
+                ]
```

Comment:
Same as above, the end of the method call can be on the same line as the closing array bracket.